db/entity/src/conversions/logs.rs

```diff
@@ -0,0 +1,69 @@
+use sea_orm::Set;
+
+use hopr_crypto_types::types::Hash;
+use hopr_primitive_types::prelude::*;
+
+use crate::{log, log_status};
+
+impl From<SerializableLog> for log::ActiveModel {
+    fn from(value: SerializableLog) -> Self {
+        log::ActiveModel {
+            address: Set(value.address),
+            topics: Set(value.topics.join(",")),
+            data: Set(value.data),
+            block_number: Set(value.block_number.to_be_bytes().to_vec()),
+            transaction_hash: Set(Hash::from_hex(value.tx_hash.as_str())
+                .expect("invalid tx_hash")
+                .as_ref()
+                .to_vec()),
+            transaction_index: Set(value.tx_index.to_be_bytes().to_vec()),
+            block_hash: Set(Hash::from_hex(value.block_hash.as_str())
+                .expect("invalid block_hash")
+                .as_ref()
+                .to_vec()),
+            log_index: Set(value.log_index.to_be_bytes().to_vec()),
+            removed: Set(value.removed),
+            ..Default::default()
+        }
+    }
+}
+
+impl From<log::Model> for SerializableLog {
```

Comment:
Consider making this a `TryFrom` instead.

This way, if I corrupt the table in the DB, I will panic the entire application.